x-pack/plugins/secops/public/components/loading/index.tsx

```diff
@@ -40,4 +40,5 @@ export const InfraLoadingStaticContentPanel = styled.div`
   flex: 0 0 auto;
   align-self: center;
   text-align: center;
+  height: fit-content;
```

Comment:
Do we have to support IE11 really?

Because i think some CSS 3 such as `fit-content` won't work?

https://caniuse.com/#search=fit-content

Unless I'm missing a a polly-fill for it that we have or style components does something additional. We might have to have a discussion and trial run of IE-11 if we are to support it for version 1.0 or not at our next team meeting.